docs/configuration.md

```diff
@@ -158,22 +158,23 @@ send events to Porta.
 ### CONFIG_MASTER_SERVICE_ID
 
 - The service ID of the master account in Porta.
-- Optional. Defaults to 1.
-- Applies to: worker.
+- Optional. If not provided, Apisonator does not report metrics to the master
+account of Porta.
```

Comment:
I think we might want to emit a warning about this in the logs.

Reply:
I'll answer the 2 questions here.

If a worker picks up a job and it does not have a master service ID configured, it logs an error and notifies it with the external error manager configured (bugsnag, etc.). That was included in the previous PR linked above. That case shouldn't be common though. If the listener has a master service ID configured and the worker does not, it means that something is not configured correctly.

About emitting a warning when the master service ID, I don't think it's a good idea. The reason is that there are many customers with this configuration already. If we change this they'll suddenly start seeing some warnings about a feature they didn't even know existed. I think we should emit a warning only if product decides that this feature should be enabled by default in following versions.

Reply:
I think I did not convey the context correctly, so here I meant to raise a warning in the worker if a job is found with no master id configured, not in the listener, since that would depend on whether this becomes mandatory or not.

> it means that something is not configured correctly

For example, upgrading from a previous release where the listener's previous version enqueued the job and the workers new version picks it up without a configured master svc id. We could have this to signal a problem with a fully configured and upgraded release, although there is this window of time in which in-flight jobs could give this warning (I don't think investing in solving that is worth it).

> If a worker picks up a job and it does not have a master service ID configured, it logs an error and notifies it with the external error manager configured (bugsnag, etc.).

So that's what I meant, thanks.